### PR TITLE
no more vars (convert var to let/const)

### PR DESCRIPTION
**Pull request by raphamorim**
After discuss with @gaearon in https://github.com/facebook/react/pull/11637. I guess it's best to continue with https://github.com/facebook/react/pull/6097.

I took the liberty of following the footsteps of #6097 and sending this PR. 
But still missing run it on files with Flow.

### CODE REVIEW

File: packages/react-art/npm/Circle.js [hunk 1]
```diff
@@ -17,31 +17,31 @@
 
 'use strict';
 
-var assign = require('object-assign');
-var PropTypes = require('prop-types');
-var React = require('react');
-var ReactART = require('react-art');
+const assign = require('object-assign');
```
Comment (gaearon) [inline packages/react-art/npm/Circle.js]: We can't do this. This file ships on npm without transpilation. So it needs to be ES5. Same for all other files in `/npm/` folder.

Comment (raphamorim) [reply packages/react-art/npm/Circle.js]: Ok, I'll change it.

**Comment by raphamorim** [comment]
Ok, leave it to me @gaearon :octocat:

File: packages/react-art/npm/Circle.js (hunk 1 above)
Comment (raphamorim) [reply packages/react-art/npm/Circle.js]: changed in 4446de5

**Comment by raphamorim** [comment]
@gaearon I was wondering if I could send the remaining files in a separate PR. The reason is the remaining files I'm editing after the command (e.g: server-side tests). 
Btw: the conflicts of that PR is already quite large (due to the number of files changed).

**Comment by gaearon** [comment]
This is really hard to review too.
Can you send a separate PR for each package please?

**Comment by raphamorim** [comment]
Sure.

**Pull request closed without merging**